### DISCUSSION TITLE
backmapping atp from cg to aa

### DISCUSSION THREAD

**Discussion by Nancyarvin** [Martini-Force-Field-Initiative/MartiniDiscussions#21] [Q&A]
Hi, I am trying to back map ATP from CG-AA.
But, the output (without minimization) I am getting is not correct, I am not able to figure out the issue. I am attaching all the necessary files in the google drive link attached here. The output structure is somewhat like this:
![image](https://github.com/user-attachments/assets/13331735-e180-409f-acd8-2ced78a2e129)
Any help would be highly appreciated, Thanks!
[https://drive.google.com/drive/folders/15uMKbUxuMZLCDueddCdW_XjP2IZcl3G8?usp=sharing](url)

**Comment by yuqingxiong96** [comment]
You should minimize this by noting define=-DFLEXIBLE, which allows the intramolecular bond length, Angle, and twist Angle to change in the simulation based on the parameters in the top file
